### PR TITLE
Fix sonarlint vulnerability

### PR DESCRIPTION
**Pull request by Kerruba**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import io.fabric8.zjsonpatch.internal.guava.Strings;
+import org.entando.kubernetes.model.debundle.EntandoDeBundle;
+
+public class BundleUtilities {
+
+    public static String getBundleVersionOrFail(EntandoDeBundle bundle, String version) {
+       
+        String versionToFind = version;
+        if ( ! (versionToFind.matches("\\d+\\.\\d+") || versionToFind.matches("\\d+\\.\\d+\\.\\d+"))) {
```
Comment (joewhite101) [inline src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java]: This needs a comment. What is happening here?

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-component-manager/pull/12#pullrequestreview-371202040" title="Approved">joewhite101</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-component-manager).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-component-manager/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java (hunk 1 above)
Comment (Kerruba) [reply src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java]: Basically this is to test support both versions formatted as versions - e.g. 2.0, 3.1.1, etc - and "meta" versions which are already included in npm - i.e. "latest" version

**Pull request merged**